app/src/components/project/PythonProjectGuide.tsx

```diff
@@ -104,19 +119,31 @@ export function PythonProjectGuide(props: PythonProjectGuideProps) {
       <View paddingBottom="size-100">
         <Text>
           <b>arize-phoenix-otel</b> automatically picks up your configuration
-          from environment variables (e.x. <b>PHOENIX_CLIENT_HEADERS</b> for
-          authentication). See{" "}
+          from
           <ExternalLink href={PHOENIX_ENVIRONMENT_VARIABLES_LINK}>
             environment variables
           </ExternalLink>
         </Text>
       </View>
-      {isHosted ? (
-        <CodeWrap>
-          <PythonBlockWithCopy
-            value={HOSTED_PHOENIX_ENVIRONMENT_VARIABLES_PYTHON}
-          />
-        </CodeWrap>
+      <CodeWrap>
+        <PythonBlockWithCopy value={environmentVariablesPython} />
+      </CodeWrap>
+      {isAuthEnabled ? (
+        <View paddingBottom="size-100" paddingTop="size-100">
+          <IsAdmin
+            fallback={
+              <Text>
+                Personal API keys can be created and managed on your{""}
+                <ExternalLink href="/profile">Profile</ExternalLink>
```

Comment:
these aren't external links are they? not sure if it matters here

Reply:
It's an external link from the perspective that it opens a new tab rather than navigating within the SPA